datasource/netcdf_groundwater.py

```diff
@@ -90,49 +100,28 @@ def ds(self):
                 self._ds = h5py.File(self.file_path, "r")
             except IOError as e:
                 logger.error(e)
```

Comment:
```suggestion
                logger.exception(e)
```

logger.exception is hetzelfde als logger.error, maar als extraatje print 'ie de traceback ook nog eens.